app/src/main/AndroidManifest.xml

```diff
@@ -12,7 +12,12 @@
     <uses-permission
         android:name="android.permission.WRITE_EXTERNAL_STORAGE"
         tools:ignore="ScopedStorage" />
-    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+    <uses-permission
+        android:name="android.permission.READ_EXTERNAL_STORAGE"
+        android:maxSdkVersion="32" />
```

Comment:
how to request external storage permission for 33?

Reply:
For 33 API you cannot request permission for external storage. You can only use scoped storage of your application,
Or if you want to pick image, you can use new MediaPicker with selecting media type

Reply:
If you need to read audio, declare also the android.permission.READ_MEDIA_AUDIO. For video, use the android.permission.READ_MEDIA_VIDEO permission. If the user has already given his permission to READ_EXTERNAL_STORAGE, the newly declared media permissions will automatically be granted when he updates his app to the new target level.
@1sakov